AGXUnity/BrickUnity/BrickUtils.cs

```diff
@@ -22,8 +22,16 @@ public static void SetupBrickEnvironment()
     {
       if (string.IsNullOrEmpty(System.Environment.GetEnvironmentVariable("BRICK_DIR")) && string.IsNullOrEmpty(System.Environment.GetEnvironmentVariable("BRICK_MODULEPATH")))
       {
-        var modulePath = System.IO.Path.Combine(Application.dataPath, "AGXUnity", "Plugins", "x86_64", "Brick", "modules");
-        System.Environment.SetEnvironmentVariable("BRICK_MODULEPATH", modulePath);
+        if (Application.isEditor)
+        {
+          var modulePath = System.IO.Path.Combine(Application.dataPath, "AGXUnity", "Plugins", "x86_64", "Brick", "modules");
+          System.Environment.SetEnvironmentVariable("BRICK_MODULEPATH", modulePath);
+        }
+        else
+        {
+          var modulePath = System.IO.Path.Combine(Application.dataPath, "brick", "modules");
```

Comment:
small `b` in brick in this new "non-editor" path, but large `B` in the "editor-path".

Should we be consistent? or is is the small b due to some other consistensy?

Reply:
It does not really matter, since we create that folder. The small letter comes from simply bringing in code from another project where it was small. Can change it to be consistent.

Reply:
The entire Brick folder in the project should not be copied, only the modules folder, since all the Brick dlls are in there also, and they are copied automatically when building